models/staging/stg_dbt__model_executions.sql

```diff
@@ -17,7 +17,7 @@ dbt_run as (
 
     select *
     from run_results
-    where data:args:which = 'run'
+    where data:args:which in ('run', 'build')
```

Comment:
On implication of this is that any tests or seeds executed as part of the `build` will also appear in the model executions model. At the moment the only filter between results which are tests, models or seeds is this command filter.

My suggestion is that we join on the manifest to get the differences. I'll put together a separate PR to show how that might be possible.